### PR TITLE
Added as default to use all opals

### PR DESCRIPTION
**Pull request by timcadman**

### CODE REVIEW

File: R/make-outcome.R [hunk 1]
```diff
@@ -492,7 +492,7 @@ to have a shorter name.",
     "created containing the following variables:\n\n"
   )
 
-  message(data_available)
+  print(data_available)
```
Comment (sidohaakma) [inline R/make-outcome.R:495]: Why use print here?

**Review by sidohaakma — CHANGES_REQUESTED** [review]

File: R/make-outcome.R (hunk 1 above)
Comment (timcadman) [reply R/make-outcome.R:495]: If I use "message" it doesn't print the tibble, instead it prints, e.g. "c(var_name_1, var_name_2). I think there's something about tibbles that mean you need to use "print". If I'm getting this wrong though do say.

Comment (sidohaakma) [reply R/make-outcome.R:495]: I think you can use print in the message context. Something like this message(print(tibbleobject)).

**Review by sidohaakma — APPROVED** [review]
I am going to merge it as is. I could not find a solution to not print the overhead in the message()

**Pull request merged**